trestle/core/control_io.py

```diff
@@ -1125,7 +1137,8 @@ def read_implemented_requirement(
 
         imp_req.statements = list(statement_map.values())
         ControlIOReader._insert_header_content(imp_req, header, control_id)
-        return imp_req
+        sort_id = header.get('sort-id', control_id)
```

Comment:
Question, if we have the `sort-id` in the control properties and the yaml header, which one will win? Is it also controlled by `override-header-values` flag ?

Reply:
That function happens to be called only during ssp-assemble where no yaml header is provided.  So the sort-id only comes from the markdown header.  None of the assemble tools allow providing a yaml header, so it will always come from the markdown.

I changed write_control so that it explicitly handles sort-id according to overwrite-header-values in a consistent manner.  If it is already in the markdown control it will not get changed - but otherwise if it is not present it will be installed.